FLACONI.md

```diff
@@ -25,32 +25,35 @@ git merge -S n0rad/master
 
 ## Build release artifacts
 
-For git tagging, use the same tag name as integrations/terraform-=provier-github has with a `-flaconi` suffix.
+When creating a new GitHub release, use the same release name (and git tag) as
+integrations/terraform-provider-github is using for its latest. We're simply enhancing it
+with n0rad's addition and want to follow their version scheme.
 
-When creating a new GitHub release, use the same release name as integrations/terraform-provider-github
-is using for its latest. We're simply enhancing it with n0rad's addition and want to follow their
-version scheme.
-
-1. Create git tag with `v[0-9]\.[0-9]\.[0-9]-flaconi` (e.g.: `v4.19.0-flaconi`)
-2. Create a GitHub Release with name: `v[0-9]\.[0-9]\.[0-9]` (e.g.: `v4.19.0`)
-3. Create `release/` directory
+1. Delete latest tag from `integrations/terraform-provider-github` (we're overwriting it)
+   ```bash
+   git tag -d v4.19.0
+   git push --delete origin  v4.19.0
+   ```
+2. Create git tag with `v[0-9]\.[0-9]\.[0-9]` (e.g.: `v4.19.0`)
```

Comment:
I don't get it, why not creating a new release but replacing the old one ?

Reply:
As an example:

Currently the latest release with inteagrations/terraform-provider-github is `v4.19.0`. We could now create our own release tag with a later versoin, e.g: `v4.20.0`. However this tag is now not in sync with upstream provider anymore and also misleading. Especially when upstream is now also creating a tag `v4.20.0`.

The goal here is to be fully in sync with upstream, just with the addition of n0rad.